packages/frontend/src/components/navigation/NavLinks.js

```diff
@@ -146,7 +146,7 @@ const NavLinks = () => (
             }
             target='_blank' 
             className="usn-button"
-            onClick={() => Mixpanel.track('Click Buy USN')}
+            onClick={() => Mixpanel.track('Click Buy USN')} rel="noreferrer"
```

Comment:
```suggestion
            onClick={() => Mixpanel.track('Click Buy USN')}
            rel="noreferrer"
```

Reply:
Heh didn't even notice, I only included it to fix the lint break 🙄